server/kv.go

```diff
@@ -46,7 +45,7 @@ type SecretsStore interface {
 }
 
 type InstanceStore interface {
-	CreateInactiveCloudInstance(types.ID) error
+	CreateInactiveCloudInstance(_ types.ID, actingUserID string) error
```

Comment:
should we simplify the naming (and add a comment to the effect) - can we create an "active" cloud instance?